koku/api/iam/test/tests_serializers.py

```diff
@@ -115,14 +112,21 @@ def test_update_user(self):
         # update the user
         current_user = User.objects.get(pk=1)
         new_data = {'username': current_user.username,
-                    'password' : 's3kr!t'}
+                    'password': 'n3w P4sSw0Rd',
+                    'first_name': 'Wade',
+                    'last_name': 'Wilson'}
         new_serializer = UserSerializer(current_user,
                                         new_data)
         if new_serializer.is_valid(raise_exception=True):
             new_serializer.save()
 
         # test the update
-        self.assertEqual(new_data['password'], User.objects.get(pk=1).password)
+        self.assertEqual(new_data['password'],
+                         User.objects.get(pk=1).password)
```

Comment:
We need to be really careful pulling records on default numeric primary keys if we switch to testing with Postgres, sequences are just going to bite us. It's also not the most reliable way of querying for a specific record in a table.